### PR TITLE
#7 add introduction

### PR DESCRIPTION
**Pull request by jakubiec**

### CODE REVIEW

File: ddd-koans/Introduction/About course/task.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <br/>
+    <br/>
+    You will learn how to apply Domain-Driven Design tactical building blocks using Kotlin.
+    <br/>
+    <br/>
+    In this course, we would like to focus on the domain of music.
+    <br/>
+    <br/>
+    No worries, you don't need to have an understanding of music theory.
+    <br/>
```
Comment (JacekDubikowski) [inline ddd-koans/Introduction/About course/task.html:13]: Why is this the only place where is only one <br> markup?

File: ddd-koans/Introduction/About course/task.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<html>
```
Comment (JacekDubikowski) [inline ddd-koans/Introduction/About course/task.html:1]: Should we consider adding a table of contents here in form of a simple list?

File: ddd-koans/Introduction/About course/task.html (hunk 1 above)
Comment (jakubiec) [reply ddd-koans/Introduction/About course/task.html:13]: just visually it looked better for me

File: ddd-koans/Introduction/About course/task.html (hunk 2 above)
Comment (jakubiec) [reply ddd-koans/Introduction/About course/task.html:1]: good idea, will do it!

**Review by JacekDubikowski — APPROVED** [review]

File: ddd-koans/Introduction/About course/task.html (hunk 2 above)
Comment (jakubiec) [reply ddd-koans/Introduction/About course/task.html:1]: done

File: ddd-koans/Introduction/About course/task.html (hunk 1 above)
Comment (jakubiec) [reply ddd-koans/Introduction/About course/task.html:13]: done

**Pull request merged**